services/QuillLMS/spec/workers/rematch_updated_questions_worker_spec.rb

```diff
@@ -25,16 +41,16 @@
 
     expect(HTTParty).to receive(:post).with(
       "test_url",
-      body: {type: 'questions', uid: 'prod_question'}.to_json,
+      body: {type: 'questions', uid: 'prod_question', delay: 0}.to_json,
       headers:  {'Content-Type' => 'application/json', 'Accept' => 'application/json'}
     ).once
 
     expect(HTTParty).to receive(:post).with(
       "test_url",
-      body: {type: 'questions', uid: 'beta_question'}.to_json,
+      body: {type: 'questions', uid: 'prod_question2', delay: 7}.to_json,
       headers:  {'Content-Type' => 'application/json', 'Accept' => 'application/json'}
     ).once
 
```

Comment:
Is it worth checking that `uid: 'prod_question3'` is _not_ received by the endpoint?

Reply:
So this test will fail if the `HTTParty` is called without the specified arguments, so this is covered. I could explicit make a not expect, but that catchall is more thorough. Screenshot of when  `prod_question3` is called in the test.
<img width="1051" alt="Screen Shot 2021-05-11 at 6 09 26 PM" src="https://user-images.githubusercontent.com/1304933/117890979-215b7080-b284-11eb-90ed-310b937edffe.png">